From 59e7ba1bf0d63ddcd8e3d40888bcac0b7223e70e Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.xyz>
Date: Thu, 28 Dec 2023 22:26:57 -0800
Subject: [PATCH 14/39] grub-core: drop GRUB title from the menu

We did not display it before, and it's really not very useful since we don't
encourage reinstalling GRUB anyway.
---
 grub-core/normal/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/grub-core/normal/main.c b/grub-core/normal/main.c
index bd4431000..8b8565f24 100644
--- a/grub-core/normal/main.c
+++ b/grub-core/normal/main.c
@@ -209,7 +209,7 @@ grub_normal_init_page (struct grub_term_output *term,
 
   grub_term_cls (term);
 
-  msg_formatted = grub_xasprintf (_("GNU GRUB  version %s"), PACKAGE_VERSION);
+  msg_formatted = grub_calloc(1, 8); /* empty message */
   if (!msg_formatted)
     return;
 
-- 
2.47.0

